nuxt.config.ts

```diff
@@ -174,12 +174,16 @@ const config: Configuration = {
     dir: 'public',
     routes: (function () {
       const events = fs.readdirSync(path.resolve(__dirname, 'content', 'events'))
-        .filter(filename => path.extname(filename) === '.md')
+        .filter(isMarkdownContent)
```

Comment:
The way to get the events and the experiments is the same, only changing the string `events` to `experiments`. Could you create a function with the functionality in common?

Reply:
I considered it non related to this PR but I will do when adressing the other one.